server/src/main/java/org/opensearch/rest/action/admin/cluster/RestClusterStatsAction.java

```diff
@@ -54,7 +63,34 @@ public class RestClusterStatsAction extends BaseRestHandler {
 
     @Override
     public List<Route> routes() {
-        return unmodifiableList(asList(new Route(GET, "/_cluster/stats"), new Route(GET, "/_cluster/stats/nodes/{nodeId}")));
+        return unmodifiableList(
+            asList(
+                new Route(GET, "/_cluster/stats"),
+                new Route(GET, "/_cluster/stats/nodes/{nodeId}"),
+                new Route(GET, "/_cluster/stats/{metric}/nodes/{nodeId}"),
+                new Route(GET, "/_cluster/stats/{metric}/{index_metric}/nodes/{nodeId}")
```

Comment:
it should be nodes/nodeId/{metric} right, same for the other one as well?

Reply:
From https://github.com/opensearch-project/OpenSearch/pull/15938#discussion_r1766842520, we decided to have path as  /stats/{metric}/nodes instead on /nodes/nodeId/{metric}, so that /stats/ is not a dummy path.